pkg/kubectl/cmd/annotate.go

```diff
@@ -85,6 +85,7 @@ func NewCmdAnnotate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Long:    annotate_long,
 		Example: annotate_example,
 		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckDeprecatedGloablFlags(cmd)
```

Comment:
Is there a better way to deprecate global flags other than calling it over and over in each kubectl command?